embossc

```diff
@@ -59,48 +57,28 @@ def _parse_args(argv):
 
 
 def main(argv):
-
   flags = _parse_args(argv)
   base_path = os.path.dirname(__file__) or "."
-  subprocess_environment = os.environ.copy()
+  sys.path.append(base_path)
```

Comment:
Can the imports be moved to the top? Afaict these modules are imported elsewhere without updating the path.

Reply:
Unfortunatly not. `embossc` is the entry point for end-users of the `emboss`, for example:
```bash
python3 ~/workspace/emboss/embossc
  --output-path /tmp \
  --import-dir public \
  public/pw_bluetooth/hci_events.emb
```
The above will fail if we move the imports to before we set up the python path. 

We can see the previous did essentially the same thing, just with `PYTHONPATH` instead:
```
  if subprocess_environment.get("PYTHONPATH"):
    subprocess_environment["PYTHONPATH"] = (
      base_path + ":" + subprocess_environment.get("PYTHONPATH"))
  else:
    subprocess_environment["PYTHONPATH"] = base_path
```